spec/e2e/pages/short-form/demographic-survey.coffee

```diff
@@ -33,8 +35,8 @@ class DemographicSurvey extends AngularPage
   fill: (opts = {}) ->
     @userGender.sendKeys(@defaults.userGender)
     @genderOther.clear().sendKeys(@defaults.genderOther)
-    # Don't fill in sexual orientation so this form isn't skipped
-    # TODO: when making change to never skip demographic survey, add sexual orientation changes back.
+    @userSex.sendKeys(@defaults.userSex)
```

Comment:
I think you need to add "userSex" and "userSexOther" to the `fields` array in `expectToMatch` too. Hard to tell from the diff but I removed them in my previous PR: https://github.com/SFDigitalServices/sf-dahlia-web/pull/1381/files#diff-8473ca875d6ef1e7536aaadac4eef866L42